Service.Core/Kernel/Kernel.Test/TestUtils/WaitForTime.hpp

```diff
@@ -4,8 +4,8 @@
 
 /**
 * @brief class waiting for a certain amount of time.
-* @details Emulates the "normal behavior" of client-server interaction. /
-*          Used in testing to temporarily process actions on the client/server side. /
+* @details Emulates the "normal behavior" of client-kernel interaction. /
```

Comment:
Simulates the "normal behavior" of the user when interacting with services. It is used in testing to create delays in work on either side

Reply:
Done